README.md

```diff
@@ -4,6 +4,7 @@
 
 This project allow to allocate mac addresses from a pool to secondary interfaces using
 [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
+It also allocates mac addresses to following primary interfaces: Multus-cni, Slirp and Masquerade.
```

Comment:
Multus-cni is not a primary interface. We should say we allocate for secondary interfaces. Then say that for VirtualMachines, we also allocate for the primary interface if they are using masquerade or slirp binding mechanism. (and link to kubevirt.io docs regarding these settings)